core/server/master/src/main/java/alluxio/web/WebInterfaceConfigurationServlet.java

```diff
@@ -37,19 +35,19 @@
 @ThreadSafe
 public final class WebInterfaceConfigurationServlet extends HttpServlet {
   private static final long serialVersionUID = 2134205675393443914L;
-  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
-  private static final Set<String> ALLUXIO_CONF_EXCLUDES = Sets.newHashSet(
-      PropertyKey.MASTER_WHITELIST.toString());
 
   private final FileSystemMaster mFsMaster;
+  private final transient MasterProcess mMasterProcess;
```

Comment:
any particular reason to have `MasterProcess` `transient`? If yes, please java doc it.

Reply:
I put List<ConfigProperty> getConfiguration() method inside AlluxioMasterProcess.java. 

All the WebServlet which use MasterProcess make it transient.
Maybe this is because that info does not need to transport over the network but directly shows on the Master web address.

Reply:
servlets are never serialized, so there is no reason for any of their fields to be `transient`

Reply:
We do not want servlets to be serialized, but our web servlets implement HttpServlet, and HttpServlet implements Serializable. That may be the reason why previous contributors all want to make those fields transient.

Reply:
Oh you're right! We should make `mFsMaster` transient as well in case the servlet engine ever tries to serialize our servlets